CMakeLists.txt

```diff
@@ -203,6 +203,15 @@ add_custom_command(
 
 add_custom_target(phony_always_runs ALL DEPENDS ${CMAKE_BINARY_DIR}/${_script_file})
 
+add_custom_target(
+    generate_version_header
+    COMMAND ${CMAKE_SOURCE_DIR}/scripts/version_header.sh ${LANTERNDB_VERSION} ${CMAKE_BINARY_DIR}
```

Comment:
`cmake` has a built-in function for this kind of stuff (`configure_file`).
Is there a reason we are not using that?
(See `CONTROL_TEMPLATE` in this CmakeLists.txt for example usage)